test/use-swr-cache.test.tsx

```diff
@@ -426,3 +386,115 @@ describe('useSWR - cache provider', () => {
     expect(createCacheProvider).toBeCalledTimes(1)
   })
 })
+
+describe('useSWR - global cache', () => {
+  it('should return the global cache and mutate by default', async () => {
+    let localCache, localMutate
+    function Page() {
+      const { cache, mutate } = useSWRConfig()
+      localCache = cache
+      localMutate = mutate
+      return null
+    }
+
+    render(<Page />)
+    expect(localCache).toBe(SWRConfig.default.cache)
+    expect(localMutate).toBe(globalMutate)
+  })
+
+  it('should be able to update the cache', async () => {
+    const fetcher = _key => 'res:' + _key
+    const keys = [createKey(), createKey()]
+
+    let cache
+    function Section() {
+      const [index, setIndex] = useState(0)
+      cache = useSWRConfig().cache
+      const { data } = useSWR(keys[index], fetcher)
+
+      return <div onClick={() => setIndex(1)}>{data}</div>
+    }
+
+    function App() {
+      return <Section />
+    }
+
+    const { container } = render(<App />)
+    await screen.findByText(fetcher(keys[0]))
+
+    expect(cache.get(keys[1])).toBe(undefined)
+    fireEvent.click(container.firstElementChild)
+    await act(() => sleep(10))
+
+    expect(cache.get(keys[0])).toBe(fetcher(keys[0]))
+    expect(cache.get(keys[1])).toBe(fetcher(keys[1]))
+  })
+
+  it('should correctly mutate the cached value', async () => {
+    const key = createKey()
+    let mutate
+
+    function Page() {
+      const { mutate: mutateWithCache } = useSWRConfig()
+      mutate = mutateWithCache
+      const { data } = useSWR(key, null)
+      return <div>data:{data}</div>
+    }
+
+    function App() {
+      return <Page />
+    }
+
+    render(<App />)
+    screen.getByText('data:')
+    await act(() => mutate(key, 'mutated value', false))
+    await screen.findByText('data:mutated value')
+  })
+
+  it('should work with revalidateOnFocus', async () => {
+    const key = createKey()
+    let value = 0
+    function Foo() {
+      const { data } = useSWR(key, () => value++, {
+        dedupingInterval: 0
+      })
+      return <>{String(data)}</>
+    }
+    function Page() {
+      return <Foo />
+    }
+    render(<Page />)
```

Comment:
Or just rename Foo -> Page

```suggestion
    render(<Foo />)
```